src/node/stage/approved.rs

```diff
@@ -370,18 +396,22 @@ impl Approved {
         }
 
         let conn_infos: Vec<_> = self
-            .chain
+            .shared_state
+            .sections
```

Comment:
`our_elders` shall be a direct function of the shared_state ?

Reply:
We used to have small delegating functions like that, but I removed them because I saw no point in having them when we can access the underlying container (`sections`) directly.